Behavior/Resources/Localizable.xcstrings

```diff
@@ -328,6 +329,7 @@
       }
     },
     "NOTIFICATION_PERMISSIONS_DESCRIPTION" : {
+      "extractionState" : "stale",
```

Comment:
Please remove all stale localization values.

If you want to learn more about localization, please check out this video for more details: https://developer.apple.com/wwdc23/10155.